nextcloudappstore/core/forms.py

```diff
@@ -1,8 +1,10 @@
+from django.contrib.auth.models import User
```

Comment:
Would be nice to get the user model from the settings, but I'm not sure how to do that since `settings.AUTH_USER_MODEL` is a string.